spec/acceptance/sensu_check_spec.rb

```diff
@@ -54,6 +54,32 @@
     end
   end
 
+  context 'with chunk size' do
+    it 'should work without errors' do
+      pp = <<-EOS
+      class { '::sensu::backend':
+        sensuctl_chunk_size => 1,
+      }
+      sensu_check { 'test3':
+        command       => 'check-http3.rb',
+        subscriptions => ['demo'],
+        handlers      => ['email'],
+        interval      => 60,
+      }
+      sensu_check { 'test4':
+        command       => 'check-cpu4.rb',
+        subscriptions => ['demo'],
+        handlers      => ['email'],
+        interval      => 60,
+      }
+      EOS
+
+      # Run it twice and test for idempotency
+      apply_manifest_on(node, pp, :catch_failures => true)
+      apply_manifest_on(node, pp, :catch_changes  => true)
```

Comment:
should this test verify that chunk setting matches `1` ?

Reply:
We can't, acceptance tests execute the real commands so what we get back is all resources and no way to verify chunking was used since end result is the same as not using chunking.  The unit tests are what ensure we're actually adding the `--chunk-size` flag but no way to verify this with acceptance tests.

Reply:
The idea behind using 1 was to ensure that even really small chunk size returns all records which is done because the catalog contains more than 1 resource and if chunking returned just 1 then it wouldn't be idempotent as Puppet would keep wanting to add one resource.